packages/ui/src/post-maker/TweetGenAI/TweetGenAI.tsx

```diff
@@ -182,16 +187,13 @@ export const TweetGenAI = ({ content }: TweetGenAIProps) => {
         </Box>
       ) : (
         <Stack spacing={4}>
-          {generatedPosts?.map((genPost: string, idx: number) => {
-            // TODO: Use PostSentenceCreator component
+          {generatedPosts?.map((genPost: string) => {
             return (
-              <Textarea
-                name={`aiPost-${idx}`}
-                id={`aiPost-${idx}`}
-                key={idx}
-                value={genPost}
-                onChange={e => console.log(e.target.value)}
-                bg={'whiteAlpha.700'}
+              <PostSentenceCreator
+                key={postId}
+                initialContent={genPost}
+                hashtagId={hashtag.id}
+                postId={postId}
```

Comment:
We can apply purple color for PostSentenceCreator only in TweetGenAI component.

```suggestion
                postId={postId}
                colorScheme={'purple'}
```